docs/index.md

```diff
@@ -50,6 +50,13 @@ Follow the following steps to get set up:
 
 1. Install necessary dependencies above.
 
+    ```sh
+    sudo apt-get install libprotobuf-dev
+    sudo apt-get install libprotobuf-compiler
+    sudo apt-get install libboost-all-dev
```

Comment:
These three lines can be combined in to one:
sudo apt-get install libprotobuf-dev libprotobuf-compiler libboost-all-dev